docker-compose.yml

```diff
@@ -51,6 +51,7 @@ services:
     ports:
       - '3000:3000'
     environment:
+      - NEXT_PUBLIC_API_URL=http://localhost:4000/api/v1
```

Comment:
Will this work when deployed? Should we create an environment variable?

Reply:
oh yeah this is going to need to be added to TF, thanks

Reply:
This will be dynamic based on what cloudfront sets. I think LTC has a way to set this based on the environment.